shanoir-ng-front/src/app/studies/study/study.component.ts

```diff
@@ -201,6 +201,7 @@ export class StudyComponent extends EntityComponent<Study> {
             'profile': [this.study.profile, [Validators.required]],
             'withExamination': [this.study.withExamination],
             'clinical': [this.study.clinical],
+            'publicDescription': [this.study.publicDescription, [Validators.maxLength(255)]],
```

Comment:
255 Seems quite low for a public description no ?

Reply:
Hi Pierre-Henri, I agree with JCD. As a description, I would just name "(study) description" (the flag is making things public or not, so your publicDescription could end up in a private description). We should offer more chars and a "text area" to edit at least a small text with 3-5 lines of text. What do you think?

Reply:
It was actually based on my SQL definition that was 255 char long, we should totally update this to change the name to just 'description' and the authorized length !

Reply:
Agreed, this was based on database description (varchar (255))